beacon-chain/core/state/state.go

```diff
@@ -91,10 +93,12 @@ func GenesisBeaconState(deposits []*pb.Deposit, genesisTime uint64, eth1Data *pb
 		eth1Data = &pb.Eth1Data{}
 	}
 
+	secondsPerDay := params.BeaconConfig().SecondsPerDay
+
 	state := &pb.BeaconState{
 		// Misc fields.
 		Slot:        0,
-		GenesisTime: genesisTime,
+		GenesisTime: genesisTime - genesisTime%secondsPerDay + 2*secondsPerDay,
```

Comment:
we already do this in powchain service, we shouldn't be setting it here. Anyway we can modify the test to reflect this ?